### PR TITLE
04 sep

### PR DESCRIPTION
**Pull request by Kapil0187**

### CODE REVIEW

**Review by studentrohit123 — COMMENTED** [review]
I have reviewed your branch

File: app/views/allbookings/history.html.erb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      </thead>
+      <tbody>
+        <% @bookings.each do |booking| %>
+          <tr>
+            <th scope="row"> <%= booking.id %> </th>
+            <td> <%= booking.date %> </td>
+            <td> <%= booking.mode_type %> </td>
+            <td>
+              <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modal-<%= booking.id %>" >
+                see tickets
```
[the remark this answers is not in the document]
Comment (Kapil0187) [reply app/views/allbookings/history.html.erb:32]: there i am showing all tickets ,that's way i write name as see tickets.

File: config/routes.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@
   # get 'bookings/show_ticket', to:"bookings#show_ticket"
   resources :bookings 
   
+
```
[the remark this answers is not in the document]
Comment (Kapil0187) [reply config/routes.rb:25]: sure i will remove.

File: Gemfile [hunk 3]
```diff
@@ -72,4 +72,6 @@ gem 'dotenv-rails', groups: [:development, :test]
 
 gem "sidekiq", "~> 7.3"
 
-gem 'stripe'
\ No newline at end of file
+gem 'stripe'
+
```
[the remark this answers is not in the document]
Comment (Kapil0187) [reply Gemfile:76]: I used extra space for better readability.

File: app/controllers/payments_controller.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# app/controllers/payments_controller.rb
+class PaymentsController < ApplicationController
+  def create
+    @amount = 500
```
[the remark this answers is not in the document]
Comment (Kapil0187) [reply app/controllers/payments_controller.rb]: i will change instance to local variable

**Pull request merged**